### PR TITLE
KTOR-3940 Add isNotEmpty() and size methods to PlaceholderList

### PR DESCRIPTION
**Pull request by ustits**
**Subsystem**
ktor-server-html-builder

**Motivation**
[KTOR-3940](https://youtrack.jetbrains.com/issue/KTOR-3940)

**Solution**
Added `size` property and `isNotEmpty` method that reuse existing methods of `PlaceholderList`

### CODE REVIEW

File: ktor-server/ktor-server-plugins/ktor-server-html-builder/jvm/src/io/ktor/server/html/Template.kt [hunk 1]
```diff
@@ -36,6 +36,11 @@ public open class Placeholder<TOuter> {
 public open class PlaceholderList<TOuter, TInner> {
     private var items = ArrayList<PlaceholderItem<TInner>>()
 
+    public val size: Int
+        get() {
+            return items.size
```
Comment (rsinukov) [inline ktor-server/ktor-server-plugins/ktor-server-html-builder/jvm/src/io/ktor/server/html/Template.kt:41]: maybe convert to expression body?

**Review by rsinukov — APPROVED** [review]
LGTM, but please run `./gradlew apiDump` and commit changes.

File: ktor-server/ktor-server-plugins/ktor-server-html-builder/jvm/src/io/ktor/server/html/Template.kt (hunk 1 above)
Comment (ustits) [reply ktor-server/ktor-server-plugins/ktor-server-html-builder/jvm/src/io/ktor/server/html/Template.kt]: 👍

**Comment by ustits** [comment]
Generated api, but now build fails on codestyle which doesn't seem to be connected with this PR

**Review by rsinukov — APPROVED** [review]

**Comment by rsinukov** [comment]
Thanks for your contribution!

**Pull request merged**